### ISSUE TITLE
Bug - Page - examples failing color contrast a11y rules

### ISSUE THREAD

**Issue by nicolethoen** [patternfly/patternfly#4920]
The text in the sidebar_body is a dark text on a dark background.

![Screen Shot 2022-06-22 at 10 02 17 AM](https://user-images.githubusercontent.com/26305082/175048599-0e2ac6e7-8fbc-47a2-9564-ec413f0b533d.png)

Also, the react examples, there is some text in the page sections that are also failing color contrast checks. It seems that the default white text on the medium grey backgrounds are not contrasting enough. We should ensure that the text colors affiliated with these sections is accessible.

**Comment by Harsh3363** [comment]
I can fix this

**Comment by mcoker** [comment]
@Harsh3363 thanks! Let me know if you have any questions.

**Comment by Harsh3363** [comment]
Hii it's going to be my first contribution soo it would be great if u could help me

**Comment by mcoker** [comment]
@Harsh3363 the basic problem is that contents of the `.pf-c-page__sidebar` element are expected to be the PatternFly [navigation component](https://www.patternfly.org/v4/components/navigation), but in the examples cited in this issue, we've just placed text in the sidebar element. The navigation component looks fine when used there, but plain text does not because that isn't the intended use case and that text is using our default text color, which is effectively black.

One approach could just be to remove that text. I don't think it's 100% necessary, although I do think it's useful, so it would probably be good to keep it.

What we often do in situations like this is apply styles that are local to the website/dev workspace for specific examples to address presentational issues like this that are limited to website examples, and not problems with the PatternFly component styles. Applying that solution here, we could just write a local CSS style for these examples that makes the sidebar text white.

To see an example of local styles, look at the [button component](https://www.patternfly.org/v4/components/button/html/). We have styles for those examples that adds right and bottom margins to the buttons to create space between them.

<img width="1701" alt="Screen Shot 2022-06-28 at 4 13 06 PM" src="https://user-images.githubusercontent.com/35148959/176289056-6bd50892-f6a5-4a37-9f9c-bd9f7ab92bdf.png">

The markdown file for the button examples page is at https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/Button/examples/Button.md

Note that we have this line that imports a stylesheet just for the markdown page:

```
import './Button.css'
```

And in that same directory, you'll find [Button.css](https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/Button/examples/Button.css) with the styles that apply to the examples on the markdown page.

Using the button as an example framework, I think you could setup a local stylesheet for the [page component example markdown](https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/Page/examples/Page.md) and add a style for the sidebar to make the text white. It would probably end up looking something like:

```
.ws-core-c-page .pf-c-page__sidebar {
  color: var(--pf-global--Color--light-100);
}
```

I didn't test that, so hopefully it works! To break that down a bit:
* `.ws-core-c-page` is a class that applies to all of the examples on the page component page.
  * If you wanted to target individual examples, each example has a unique `id`, so you would use the example `id` instead (e.g., `#ws-core-c-page-vertical-nav`)
* `.pf-c-page__sidebar` - this is the element for the vertical navigation, where the text is that's failing a11y contrast.
* `--pf-global--Color--light-100` is a light version of our primary text color (`--pf-global--Color--100`) that's specifically for use against dark backgrounds.

Hopefully that gets you started, let me know if you have any questions!

**Comment by Harsh3363** [comment]
Thanks for your help  Michael I will try to fix this.

**Comment by mcoker** [comment]
Hey @Harsh3363! Just checking in to see how it's coming along. Let me know if there is anything I can do to help!

**Comment by Harsh3363** [comment]
> Hey @Harsh3363! Just checking in to see how it's coming along. Let me know if there is anything I can do to help!

Hey, @mcoker Sorry I was busy with some work but I'm working to fix this. i will let you know if I will need any help
Thanks

**Comment by mcoker** [comment]
@Harsh3363 sounds great, thanks!

**Comment by mcoker** [comment]
Howdy @Harsh3363! No rush on this, just wanted to check in and see how it's going and ask if there's anything I can do to help out. Feel free to reach out any time!

**Comment by Harsh3363** [comment]
> Howdy @Harsh3363! No rush on this, just wanted to check in and see how it's going and ask if there's anything I can do to help out. Feel free to reach out any time!

My semester end exams are going on that's why I'm not able to focus on this 
Sorry for the delay, after exams I will do it as soon as possible
